tests/test_shadow_fork.py

```
@@ -72,30 +72,18 @@ def test_create_fails_if_already_active(self, tmp_path):
-                (True, "main", ""),  # Get current branch
-                (True, "", ""),  # Check for uncommitted changes
-                (False, "", "fatal"),  # Base branch doesn't exist
-            ]
-
-            with pytest.raises(RuntimeError, match="does not exist"):
-                fork.create(base_branch="nonexistent")
-
     def test_create_warns_on_uncommitted_changes(self, tmp_path, caplog):
         """Test that create warns about uncommitted changes."""
+        import logging
+        caplog.set_level(logging.WARN)
```

Ruff (0.15.2)

[warning] Use of undocumented `logging.WARN` constant

Replace `logging.WARN` with `logging.WARNING`

(LOG009)